dist/index.js

```diff
@@ -4742,6 +4740,29 @@ function isGhes() {
     const ghUrl = new url_1.URL(process.env['GITHUB_SERVER_URL'] || 'https://github.com');
     return ghUrl.hostname.toUpperCase() !== 'GITHUB.COM';
 }
+function parseNodeVersion() {
+    let nodeVersion = core.getInput('node-version') || core.getInput('version');
```

Comment:
just wanted to ask here whether you want to check for a `falsy` value or just `null` or `undefined`? If the latter:

```suggestion
    let nodeVersion = core.getInput('node-version') ?? core.getInput('version');
```